scripts/perf-production-benchmark.mjs

```
@@ -0,0 +1,412 @@
+
+        args.markdownOutputPath = value;
+        index += 1;
+        break;
+      }
+      case '--keep-temp-cache':
+        args.keepTempCache = true;
+        break;
+      case '--help':
+      case '-h':
+        printHelp();
+        process.exit(0);
```

Biome (2.4.4)

[error] This case is falling through to the next case.

(lint/suspicious/noFallthroughSwitchClause)